ovs/lib/tests/vdisk_tests/test_get_set_config_params.py

```diff
@@ -49,7 +49,7 @@ def setUpClass(cls):
         fakesleep.monkey_patch()
         EtcdConfiguration.set('/ovs/framework/storagedriver|mds_tlogs', 100)
         EtcdConfiguration.set('/ovs/framework/storagedriver|mds_maxload', 75)
-        EtcdConfiguration.set('/ovs/framework/storagedriver|mds_safety', 2)
+        EtcdConfiguration.set('/ovs/framework/storagedriver|mds_safety', 3)
```

Comment:
Would it be wise to update the config value to 3 and leave the tests with 2 as default?